package.json

```diff
@@ -21,13 +21,12 @@
   },
   "homepage": "https://github.com/driftyco/ionic-module-template#readme",
   "devDependencies": {
```

Comment:
@ihadeed I'd vote leaving out IonicModule to keep it simple for this example, and we can document how to add it